tests/src/tools/ReadyToRun.SuperIlc/CompileDirectoryCommand.cs

```diff
@@ -57,6 +57,8 @@ class CompileDirectoryCommand
             }
 
             List<CompilerRunner> runners = new List<CompilerRunner>();
+            runners.Add(new JitRunner(null, inputDirectory.ToString(), outputDirectory.ToString(), referencePaths));
```

Comment:
I don't see a class for this in the diff. Is there a JitRunner.cs that should be in the commit?

Reply:
Of course you're right, sorry about the overlooking, should be fixed in 2nd commit together with one other overlooking (I didn't actually add support for the environment overrides to ProcessRunner even though I added them to the ProcessInfo). I'm on the verge of being able to publish the per-assembly summary statistics table, I'll mention it in this PR thread once I get it.